### PR TITLE
Add a new feature - Factorial

### PR DESCRIPTION
**Pull request by anjaliraju0097**
Factorial of a number has been added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 1. Create a program to find n fibanoc - Divya
 2. Create a program to find factorial of a number - Anjali
-3. Sum of first 50 prime numbers
```
Comment (DRD98) [inline README.md:5]: Improve the code.

**Review by DRD98 — CHANGES_REQUESTED** [review]
Kindly check the review

File: README.md (hunk 1 above)
Comment (anjaliraju0097) [reply README.md:5]: This code is fine.

**Comment by anjaliraju0097** [comment]
Check my comment.

File: README.md (hunk 1 above)
Comment (DRD98) [reply README.md:5]: Alright

**Review by DRD98 — APPROVED** [review]

**Pull request merged**